Dockerfile

```
@@ -14,9 +14,9 @@ COPY --from=ghcr.io/astral-sh/uv:0.9.1 /uv /bin/uv
 RUN : \
     # Install Pip for all other Python versions.
     && set -x \
-    && uv python install 3.8 3.9 3.10 3.11 3.12 3.13 --preview \
+    && uv python install 3.8 3.9 3.10 3.11 3.12 3.13 3.14 \
```

both python 3.8 and 3.9 reached their end of life. Let's remove them
```suggestion
    && uv python install 3.10 3.11 3.12 3.13 3.14 \
```